docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -130,14 +130,14 @@ The connector supports pushdown for a number of operations:
 * :func:`min`
 * :func:`sum`
 
-Limitations
+.. _clickhouse-sql-support:
+
+SQL support
 -----------
 
-The following SQL statements aren't  supported:
+The connector provides read access to data and metadata in
+ClickHouse.  In addition to the :ref:`globally available
```

Comment:
a ClickHouse catalog.  (It would not be everything in clickhouse - a catalog could only define 1 of N available schemas)